### ISSUE TITLE
In readme document how to disable specific rules

### ISSUE THREAD

**Issue by reagle** [ckaznocha/SublimeLinter-contrib-write-good#10]
Should one use "ignore" or "args"?

**Comment by ckaznocha** [comment]
something like this should work but I've never tested it:
```json
{
  "args": [
    "--no-passive",
    "--weasel"
  ] 
}
```

Please let me know if you have any luck and I'll update the readme or I'd happily accept a PR ;)

**Comment by reagle** [comment]
Yes, I am successfully doing this now. Could you include this in the readme?

**Comment by dylan-k** [comment]
in the readme, it would help to know, which file should contain code like the above example? where in that file should it be placed? what is the complete set of available options?

**Comment by honzik20** [comment]
Coming to this thread after a long while and I **can't** get it to work. I'm guessing that you need to add the `args` array in `SublimeLinter.sublime-settings` but this does nothing:

```
// SublimeLinter Settings - User
{
    "debug": true,
    "linters": {
        "markdownlint": {
        },
        "write-good": {
          "args": [
            "--no-passive",         
            "--no-weasel"
              ] 
            }
    },
    "show_panel_on_save": "view"
}
```

Can anyone tell me what I'm doing wrong?
